gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceClientClassComposer.java

```diff
@@ -578,6 +578,35 @@ private List<MethodDefinition> createGetterMethods(
         .collect(Collectors.toList());
   }
 
+  private static String getJavaMethod(MethodDefinition m) {
+    StringBuilder methodSignature = new StringBuilder();
+
+    // Method name
+    methodSignature.append(m.methodIdentifier().name()).append("(");
+
+    // Iterate through and add all parameters
+    List<Variable> parameters =
+        m.arguments().stream().map(VariableExpr::variable).collect(Collectors.toList());
+
+    for (int i = 0; i < parameters.size(); i++) {
+      Variable param = parameters.get(i);
+      String paramType =
+          param.type().reference() != null ? param.type().reference().name() + " " : "";
```

Comment:
Can you clarify what's going on to prevent the type from being available in some cases?

For example in the IdentityClient output:
```java
<li>createUser(String displayName, String email, age, String nickname, enableNotifications, heightFeet)
```
This should be:
```
<li>createUser(String displayName, String email, int age, String nickname, boolean enableNotifications, double heightFeet)
```

So it seems like this logic isn't able to parse primitive types, only reference types. Can this be fixed?

Reply:
```java
      String paramType = param.type().reference() == null
          ? param.type().typeKind().name().toLowerCase() // Primitive type
          : param.type().reference().name(); // Reference type
```

Reply:
Good catch. Thanks - yeah I didn't realize `reference` was only for `Object` types, and so using `typeKind` solves it if the parameter is a primitive type.